### PR TITLE
Removed archived repo remrem shared dependencies.

### PR DESCRIPTION
**Pull request by jainadc9**
### Description of the Change
Removed archived repo remrem shared dependencies.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:  <JainadC> <jainad.chinta@ericsson.com>

### CODE REVIEW

**Comment by jainadc9** [comment]
https://github.com/eiffel-community/eiffel-remrem-shared/issues/30

File: publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+     * of the protocol or service and if it is endpoint or not respectively.
+     * Example for build.gradle: manifest { attributes('remremVersionKey':
+     * 'semanticsVersion') attributes('semanticsVersion': version)
+     * attributes('isEndpointVersion': 'true') }
+     * 
+     * @return a map containing the protocol and service types with their
+     *         versions {"endpointVersions" : {"semanticsVersion" : "1.1.1"},
+     *         "serviceVersion": {"remremGenerateVersion": "0.1.1"}}
+     */
+
```
Comment (RajuBeemreddy1) [inline publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java:65]: Remove empty lines and check if the documentation is properly formatted

File: publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                                if (mainAttribs.getValue(IS_ENDPOINT_VERSION) != null) {
+                                    endpointVersions.put(versionKey, version);
+                                } else {
+                                    serviceVersion.put(versionKey, version);
+                                }
+                            }
+                        }
+                    }
+                } catch (Exception e) {
+                    // Silently ignore wrong manifests on classpath?
```
Comment (RajuBeemreddy1) [inline publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java:93]: Write some message here for failure cases

File: publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+import java.util.jar.Manifest;
+
+import com.google.gson.JsonParser;
+
+
+/**
+ * This class will search in all registered jars and their manifest file for
+ * attribute Remrem-Version-Key. It will return a list with all versions found.
+ * 
+ * @author evasiba
```
Comment (RajuBeemreddy1) [inline publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java:35]: Please check and remove if we are not keeping the author's info

**Review by RajuBeemreddy1 — CHANGES_REQUESTED** [review]

File: publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java (hunk 3 above)
Comment (jainadc9) [reply publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java:35]: Removed author's info

File: publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java (hunk 1 above)
Comment (jainadc9) [reply publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java:65]: Formatted

File: publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java (hunk 2 above)
Comment (jainadc9) [reply publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java:93]: Done

File: publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+public class VersionService {
+
+    private static final String WEB_INF = "WEB-INF";
+    private static final String VERSION = "version";
+    private static final String META_INF_MANIFEST_MF = "META-INF/MANIFEST.MF";
+    private static final String REMREM_VERSION_KEY = "remremVersionKey";
+    private static final String IS_ENDPOINT_VERSION = "isEndpointVersion";
+    private static final String ENDPOINT_VERSION = "endpointVersions";
+    private static final String SERVICE_VERSION = "serviceVersion";
+    private Logger log = (Logger) LoggerFactory.getLogger(VersionService.class);
```
Comment (dhruvin-kalavadia) [inline publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/VersionService.java:45]: Need to add import statement is for LoggerFactory

File: publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+public class VersionService {
+
+    private static final String WEB_INF = "WEB-INF";
+    private static final String VERSION = "version";
+    private static final String META_INF_MANIFEST_MF = "META-INF/MANIFEST.MF";
+    private static final String REMREM_VERSION_KEY = "remremVersionKey";
+    private static final String IS_ENDPOINT_VERSION = "isEndpointVersion";
+    private static final String ENDPOINT_VERSION = "endpointVersions";
+    private static final String SERVICE_VERSION = "serviceVersion";
+    private Logger log = (Logger) LoggerFactory.getLogger(VersionService.class);
```
Comment (dhruvin-kalavadia) [inline publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/VersionService.java:44]: Need to add import statement is for LoggerFactory

**Review by dhruvin-kalavadia — CHANGES_REQUESTED** [review]

**Review by jainadc9 — COMMENTED** [review]
Added required imports

**Review by jainadc9 — COMMENTED** [review]
Added required imports

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 - Uplifted eiffel-remrem-parent version from 2.0.6 to 2.0.7.
 - Uplifted eiffel-remrem-semantics version from 2.2.1 to 2.2.2.
 - Uplifted jackson-databind dependency to 2.13.3
+- Updated the documentation that publish doesn't validate the message.
```
Comment (dhruvin-kalavadia) [inline CHANGELOG.md:6]: Need to add point regarding removal of remrem shared dependency.

**Comment by jainadc9** [comment]
https://github.com/eiffel-community/eiffel-remrem-publish/issues/202 
Updated the documentation related to message Validation

**Review by RajuBeemreddy1 — APPROVED** [review]

**Pull request merged**